### PR TITLE
fix(migration): handle lost value last_boot, remote_addr

### PR DESCRIPTION
**Pull request by stonebuzz**
Maybe e better solution for  https://github.com/glpi-project/glpi-inventory-plugin/pull/362

Some data are lost during migration

| Table => Row  | To  | Table => row  | is migrated  |
|---|---|---|---|
| glpi_plugin_glpiinventory_inventorycomputercomputers => last_inventory_update  | =>  |  glpi_computers => last_inventory_update | :ok: |
| glpi_plugin_glpiinventory_inventorycomputercomputers => last_boot  |  =>  | glpi_computers => last_boot  | :x: |
| glpi_plugin_glpiinventory_agents => last_contact | => | glpi_agent => last_contact | :ok: 
| glpi_plugin_glpiinventory_agents => remote_addr | => | glpi_agent => remote_addr | :x: |

This PR fix this

### CODE REVIEW

**Review by trasher — APPROVED** [review]

File: install/update.native.php [hunk 1]
```diff
@@ -296,11 +296,26 @@ function pluginGlpiinventoryUpdateNative($current_version, $migrationname = 'Mig
...
                 `computers_id`,
-                `last_inventory_update`
+                `last_inventory_update`,
+                `last_boot`
               FROM `glpi_plugin_glpiinventory_inventorycomputercomputers`
           ) AS `plugin_computers` ON `plugin_computers`.`computers_id` = `computers`.`id`
           SET
-              `computers`.`last_inventory_update` = `plugin_computers`.`last_inventory_update`
+          `computers`.`last_inventory_update` = `plugin_computers`.`last_inventory_update`,
+          `computers`.`last_boot` = `plugin_computers`.`last_boot`
```
Comment (cedric-anne) [inline install/update.native.php:305]: Mybe this should be done with two different queries to be able to add a where clause:
`` WHERE `computers`.`last_inventory_update` IS NULL``
`` WHERE `computers`.`last_boot` IS NULL``

File: install/update.native.php [hunk 2]
```diff
@@ -296,11 +296,26 @@ function pluginGlpiinventoryUpdateNative($current_version, $migrationname = 'Mig
...
+        $DB->queryOrDie(
+            "UPDATE `glpi_agents` AS `agents`
+            INNER JOIN (
+              SELECT
+                `computers_id`,
+                `remote_addr`
+              FROM `glpi_plugin_glpiinventory_inventorycomputercomputers`
+          ) AS `plugin_computers` ON `plugin_computers`.`computers_id` = `agents`.`items_id` AND `agents`.`itemtype` = 'Computer'
+          SET
+          `agents`.`remote_addr` = `plugin_computers`.`remote_addr`
```
Comment (cedric-anne) [inline install/update.native.php:318]:
```suggestion
          `agents`.`remote_addr` = `plugin_computers`.`remote_addr`
          WHERE
          `agents`.`remote_addr` IS NULL OR `agents`.`remote_addr` = ''
```

File: install/update.native.php [hunk 3]
```diff
@@ -300,7 +300,35 @@ function pluginGlpiinventoryUpdateNative($current_version, $migrationname = 'Mig
...
+          WHERE `agents`.`remote_addr` IS NULL
```
Comment (cedric-anne) [inline install/update.native.php:331]:
```suggestion
          WHERE `agents`.`remote_addr` IS NULL OR `agents`.`remote_addr` = ''
```

File: install/update.native.php [hunk 4]
```diff
@@ -300,7 +300,35 @@ function pluginGlpiinventoryUpdateNative($current_version, $migrationname = 'Mig
               FROM `glpi_plugin_glpiinventory_inventorycomputercomputers`
           ) AS `plugin_computers` ON `plugin_computers`.`computers_id` = `computers`.`id`
           SET
-              `computers`.`last_inventory_update` = `plugin_computers`.`last_inventory_update`
+          `computers`.`last_inventory_update` = `plugin_computers`.`last_inventory_update`
+          ;"
```
Comment (cedric-anne) [inline install/update.native.php:304]:
```suggestion
          WHERE `computers`.`last_inventory_update` IS NULL
          ;"
```

File: install/update.native.php (hunk 2 above)
Comment (stonebuzz) [reply install/update.native.php:330]: fixed

File: install/update.native.php (hunk 4 above)
Comment (stonebuzz) [reply install/update.native.php]: fixed

**Review by cedric-anne — APPROVED** [review]

**Review by trasher — APPROVED** [review]

**Pull request merged**